### PR TITLE
refactor: update helpers

### PR DESCRIPTION
**Pull request by Plamen5kov**
based on branch: `kddimitrov/nsconfig-app-folder`
update for helper methods necessary in the CLI
related PR: https://github.com/NativeScript/nativescript-cli/pull/3398

### CODE REVIEW

File: helpers.ts [hunk 1]
```diff
@@ -52,6 +52,17 @@ export function trackDownloadProgress(destinationStream: NodeJS.WritableStream,
 	return progressStream;
 }
 
+export function isBuildFromCLI(item: string) {
```
Comment (rosen-vladimirov) [inline helpers.ts]: this function needs a return type.
Also its name is misleading. The implementation checks if the passed string contains the string `.aar` or `include.gradle`.

File: helpers.ts [hunk 2]
```diff
@@ -52,6 +52,17 @@ export function trackDownloadProgress(destinationStream: NodeJS.WritableStream,
...
+	return item.match(/.*\.aar/) ||
+			item.match(".*include.gradle");
+}
+
+export function isRecommendedAarFile(foundAarFile: string, packageJsonPluginName: string) {
```
Comment (rosen-vladimirov) [inline helpers.ts]: this function needs a return type.

File: helpers.ts [hunk 3]
```diff
@@ -52,6 +52,17 @@ export function trackDownloadProgress(destinationStream: NodeJS.WritableStream,
 	return progressStream;
 }
 
+export function isBuildFromCLI(item: string) {
+	return item.match(/.*\.aar/) ||
```
Comment (rosen-vladimirov) [inline helpers.ts]: this regex will match strings like:
* filename.aar
* .aar
* test.aar.backup

Is this the desired behavior?

File: helpers.ts [hunk 4]
```diff
@@ -52,6 +52,17 @@ export function trackDownloadProgress(destinationStream: NodeJS.WritableStream,
...
+			item.match(".*include.gradle");
```
Comment (rosen-vladimirov) [inline helpers.ts]: this regex will match strings like:
* filename.include.gradle
* include.gradle
* include.gradle.backup

Is this the desired behavior?

File: helpers.ts (hunk 3 above)
Comment (Plamen5kov) [reply helpers.ts]: yes

File: helpers.ts (hunk 4 above)
Comment (Plamen5kov) [reply helpers.ts]: yes

**Pull request merged**